src/GameMachineController.java

```diff
@@ -6,23 +7,19 @@
     private Scanner sc = new Scanner(System.in);
     private GameMachine machine = new GameMachine();
 
-    public void GameShop() {
-
-        System.out.println("Give game's name which you want buy:");
-        String wantedGame = sc.nextLine();
-        System.out.println("Pay:");
-        double payment = sc.nextDouble();
-        sc.nextLine();
-
+    public void gameShop() {
+        System.out.println("What game do you want to buy?");
         try {
-            Game game = machine.getGame(wantedGame);
-            double rest = machine.paidForGame(payment, game);
-            System.out.println("Odbierz grę " + game.getName());
-            System.out.println("Odbierz resztę " + rest);
-        } catch (NotSuchGameException e) {
-            System.err.println(e.getMessage());
-        } catch (NotEnoughtMoneyException e) {
+            String wantedGame = sc.nextLine();
+            System.out.println("Pay:");
```

Comment:
Move getting payment to the new method? Getting game name has its own method, lets do the same for payment

Reply:
I'm not sure it is what you wanted, but i make a little changes. I removed printl and but i had to leave variable payment.

Reply:
I dont see any hange. I want you to create a new method that will get payment from the user and use this new method here.